### DISCUSSION TITLE
[Feature Request] Want to start developing a mobile version

### DISCUSSION THREAD

**Discussion by sylasxu** [heroui-inc/heroui#1249] [Ideas]
### Is your feature request related to a problem? Please describe.

Stitches & nextui have great potential，But now the excellent ui libraries seem to ignore the needs of the mobile whether it is a mobile browser or a native application webview. So companies in need have to give up NextUI,  rebuild themselves or even turn to the Vue.js camp.

### Describe the solution you'd like

I'm a senior engineer trying to build the best react ui library . So I know a lot of css solutions or lib, just like Vanllia-Extract, Emotion,etc... But it seems that the performance is not as good as Stitches, so I want to join you to help build a NextUI-Mobile, build mobile components like PullRefresh, List, ActionSheet, Drawer, Picker, Navigation, FAB, etc. to enrich the competitiveness and ecology of NextUI!!

### Describe alternatives you've considered

This is a complex decision I deeply understand, and may not be agreed to, so I have to start building my own, or other platform mobile version of the component library. that's undeniably frustrating......cry

**Comment by akangaziz** [comment]
up
